utils/buttons.py

```diff
@@ -282,9 +321,14 @@ async def interaction_check(self, interaction: discord.Interaction) -> bool:
         if self.to_respond.id == getattr(interaction.user, "id", None):
             return True
 
-        await interaction.response.send_message(f"Sorry, only {self.to_respond} can respond to this prompt.", ephemeral=True)
+        await interaction.response.send_message(
+            f"Sorry, only {self.to_respond} can respond to this prompt.",
+            ephemeral=True,
+        )
+
+        return False
```

Comment:
Redundant, change the return typing to `Optional[bool]`

Reply:
Are you sure? This won't be correct typing. Other instances of interaction_check return bool

Reply:
yea i dont care, it just cares if its a falsy value